### PR TITLE
[#52527] Update documentation

### PR DESCRIPTION
**Pull request by Kharonus**
[#52527](https://community.openproject.org/work_packages/52527)

### WAT?

- add drive guide section about stop inheriting permissions

### CODE REVIEW

File: docs/system-admin-guide/integrations/one-drive/drive-guide/README.md [hunk 1]
```diff
@@ -1,12 +1,47 @@
...
 keywords: OneDrive/SharePoint file storage integration, OneDrive, SharePoint, DriveID, Azure, Drive ID
 ---
 
-# Drive ID Guide
+# Drive Guide
+
+## Configure drive for automatic management
+
+If you need a drive configured for the file storage feature "Automatically managed project folders", there are some
+previous steps to take. If the drive is destined to be used in a file storage, while the management is still based
```
Comment (cbliard) [inline docs/system-admin-guide/integrations/one-drive/drive-guide/README.md]: please clarify "management" => "permission management"

Comment (cbliard) [reply docs/system-admin-guide/integrations/one-drive/drive-guide/README.md]: and add "otherwise if" instead of just "if"

Comment (Kharonus) [reply docs/system-admin-guide/integrations/one-drive/drive-guide/README.md]: Updated.

**Comment by Kharonus** [comment]
Raised by Christophe: Should we have a redirect from `/drive-id-guide` to `drive-guide`?

**Comment by MayaBerd** [comment]
> Raised by Christophe: Should we have a redirect from `/drive-id-guide` to `drive-guide`?

@Kharonus I would say yes. @as-op could you please do this?

**Review by MayaBerd — APPROVED** [review]

**Comment by as-op** [comment]
I will add a redirect. (I have to wait for this to be merged and the website PR be merged though)

**Pull request merged**